docs/dcp2_system_design.rst

```diff
@@ -544,7 +544,7 @@ underscore, and so that a lexicographical sorting reflects both the
 hierarchical relationship between deployments, projects, datasets and snapshots
 as well as the time they were created. ::
 
-    dataset_name = "hca_" , deployment , "_" , [project_id] , "__" , creation_date , ["_" , qualifier]
+    dataset_name = [system], "_" , deployment , "_" , [project_id] , "__" , creation_date , ["_" , qualifier]
```

Comment:
I am unsure about the nomenclature here.  `project` is already an overloaded term in this doc, so I went with `system`.

Reply:
What about `atlas`? That's what Azul currently uses:

```
hannes@hannes-xps:~$ http https://service.azul.data.humancellatlas.org/index/catalogs
HTTP/1.1 200 OK
Access-Control-Allow-Headers: Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Encoding: gzip
Content-Length: 414
Content-Type: application/json
Date: Tue, 16 Nov 2021 21:13:30 GMT
Via: 1.1 a18ab1a2e56cae6e1bbaf5b1ee56db86.cloudfront.net (CloudFront)
X-Amz-Cf-Id: xqkMqtqWKcgVIItHNTp_ebFLmfMTqHckP5e7gXsqH9UahtMLDsfyrg==
X-Amz-Cf-Pop: LAX3-C4
X-Amzn-Trace-Id: Root=1-61941efa-79d3fa906f8a86d43752ce50;Sampled=0
X-Cache: Miss from cloudfront
x-amz-apigw-id: I6nHKHs-IAMFS5Q=
x-amzn-RequestId: 95785005-d3b4-450e-9bee-e02220069dc1

{
    "catalogs": {
        "dcp1": {
            "atlas": "hca",
            "internal": false,
            "plugins": {
                "metadata": {
                    "name": "hca"
                },
                "repository": {
                    "name": "tdr",
                    "sources": [
                        "tdr:broad-datarepo-terra-prod-hca2:snapshot/hca_prod_20201118_dcp1___20201209:/2"
                    ]
                }
            }
        },
        "dcp10": {
            "atlas": "hca",
```